backend/app/views/spree/admin/shared/_datepicker.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="input-group">
+  <div class="input-group-prepend">
+    <span class="input-group-text">
+      <%= icon('fa fa-calendar') %>
+    </span>
+  </div>
+  <%= f.text_field date_attr, value: datepicker_field_value(f.object.public_send(date_attr)), class: 'form-control datepicker', placeholder: placeholder_attr %>
```

Comment:
Could we add an option to set the `required` attribute?

Reply:
Would making "Available On" input required complicate the product creation process in other ways?

I was hesitant to suggest this for this PR as I think it is another can of worms.